src/constructors/css.js

```diff
@@ -3,6 +3,8 @@ import interleave from '../utils/interleave'
 import flatten from '../utils/flatten'
 import type { Interpolation, RuleSet } from '../types'
 
-export default (strings: Array<string>, ...interpolations: Array<Interpolation>): RuleSet => (
-  flatten(interleave(strings, interpolations))
-)
+export default (strings: Array<string>, ...interpolations: Array<Interpolation>): RuleSet => {
+  const rules = flatten(interleave(strings, interpolations))
+  rules.isStyledBlock = true
```

Comment:
@ramitos this is where the flow error is happening. I'm not 100% sure why you need to do this though?